### ISSUE TITLE
createMediaElementSource() requires prior call to getUserMedia() using Safari

### ISSUE THREAD

**Issue by creaktive** [mdn/webaudio-examples#56]
[media-source-buffer](https://github.com/mdn/webaudio-examples/tree/master/media-source-buffer) stays silent on Safari, unless this line is added (and the permission to use microphone is granted):

```
navigator.mediaDevices.getUserMedia({ audio: true, video: false });
```

I could not figure out whether this is a bug on Safari, or is it an undocumented behavior.

**Comment by creaktive** [comment]
Worth mentioning: even with the microphone permission granted, the sound clicks and pops so it is overall barely usable. These are known bugs, possibly unrelated to the permission: https://bugs.webkit.org/buglist.cgi?quicksearch=createMediaElementSource

**Comment by bsmth** [comment]
@Josh-Cena - is this still valid? Looks like you have context on it.

**Comment by Josh-Cena** [comment]
Yeah I can't reproduce the issue anymore, so you can assume it's fixed.

**Comment by bsmth** [comment]
Thanks, Josh. @creaktive I'm closing this one as done now, thank you.
